test/myclirc

```diff
@@ -0,0 +1,12 @@
+# vi: ft=dosini
+
+# This file is loaded after mycli/myclirc and should override only those
+# variables needed for testing.
+# To see what every variable does see mycli/myclirc
+
+[main]
+
+log_file = ~/.mycli.test.log
+log_level = DEBUG
+prompt = '\u@\h:\d> '
```

Comment:
As I think of it, maybe we can use the default prompt, expect_exact doesn't mind if the prompt starts with mycli or mariadb since you've removed it...

Reply:
You're right, I though `expect_exact` wouldn't work with the product type in the prompt but it works.
In fact it works with `less_chatty = False` now 😆 

Should I revert completely the new `tests/myclirc` file and just remove the product type from the prompt expected strings in the tests?

Reply:
I'm sure we'll need a test myclirc for future changes, so I think we can keep it this way.

Reply:
@meeuw Old prompt restored (but not deleted, still in `test/myclirc`), if you want more changes to the config file just tell me ^_^